File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BluefinTecsUserBackoffice SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BluefinTecsUserBackofficeMakeContext
{
    public static function call(array $ctxmap, ?BluefinTecsUserBackofficeContext $basectx): BluefinTecsUserBackofficeContext
    {
        return new BluefinTecsUserBackofficeContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# BluefinTecsUserBackoffice SDK utility: make_context
require_relative '../core/context'
module BluefinTecsUserBackofficeUtilities
  MakeContext = ->(ctxmap, basectx) {
    BluefinTecsUserBackofficeContext.new(ctxmap, basectx)
  }
end
